core/services/keeper/integration_test.go

```diff
@@ -166,5 +166,5 @@ func TestKeeperEthIntegration(t *testing.T) {
 	require.Equal(t, 3, len(runs))
 	prr := webpresenters.NewPipelineRunResource(runs[0])
 	require.Equal(t, 1, len(prr.Outputs))
-	require.NotNil(t, prr.Outputs[0])
+	require.Nil(t, prr.Outputs[0])
```

Comment:
This seems like it broke? Why change the test?

Reply:
Because ethtx returns `nil` result so the output looks like `[nil]`